packages/cache/README.md

```diff
@@ -37,12 +37,12 @@ A Stylis plugin or plugins that will be run by stylis during preprocessing. [Rea
 
 ### prefix: boolean | Function
 
-Allows changing Stylis' prefixing settings, this defaults to `true`. It can be a boolean or a function to dynamicly set which properties are prefixed. [More information can be found in Stylis' docs](https://github.com/thysultan/stylis.js#vendor-prefixing)
+Allows changing Stylis' prefixing settings; defaults to `true`. It can be a boolean or a function to dynamically set which properties are prefixed. [More information can be found in Stylis' docs](https://github.com/thysultan/stylis.js#vendor-prefixing).
 
 ### key: string
 
-The prefix before class names, this defaults to `css`. It will also be set as the value of the `data-emotion` attribute on the style tags that emotion inserts and it's used in the attribute name that marks style elements in `renderStylesToString` and `renderStylesToNodeStream`. This is **required if using multiple emotion caches in the same app**.
+The prefix before class names; defaults to `css`. It will also be set as the value of the `data-emotion` attribute on the style tags that emotion inserts and it's used in the attribute name that marks style elements in `renderStylesToString` and `renderStylesToNodeStream`. This is **required if using multiple emotion caches in the same app**.
```

Comment:
a semicolon inside a sentence looks slightly unnatural (at least to me), could we replace it i.e. with "which"?

Reply:
Yeah, semicolon is an unusual punctuation mark, so I can see how it looks unnatural. Using "which" there could lead to some ambiguity, so it might be better to just replace the semicolons with periods and start a new sentence.

I wonder if it would improve readability to format the option heading to incorporate the default, though?

So instead of...

> ### key: string

...format it as

> ### `key` (string, default: `"css"`)

I noticed the [babel-plugin-emotion readme](https://github.com/emotion-js/emotion/blob/master/packages/babel-plugin-emotion/README.md#sourcemap) handles this by splitting to separate lines, like this:

> ### `key`
> `string`, defaults to `"css"`

And then the rest of the description would follow in a subsequent paragraph.

Reply:
Sounds good as well, could you tweak the PR accordingly?

Reply:
Happy to. I'll follow the babel-plugin-emotion style (separating the data type and default to their own line), unless I hear otherwise from you.